components/webgpu/wgpu_thread.rs

```diff
@@ -448,7 +469,25 @@ impl WGPU {
                                 Some(render_pipeline_id),
                                 implicit)
                             );
-                            self.maybe_dispatch_wgpu_error(device_id, error);
+
+                            if let Some(sender) = sender {
```

Comment:
It would be nice to avoid the code duplication here.

Reply:
You mean duplication between render/compute pipeline? They differentiate in `CreateRenderPipelineError` on `CreateComputePipelineError` and in responses they are sending, and I think dedup would require too much complexity (generics) for not enough profit (happens only twice).

Reply:
👍